### PR TITLE
LPS-147076

### PR DESCRIPTION
**Pull request by mbowerman**
Ticket: [LPS-147076](https://issues.liferay.com/browse/LPS-147076)

Patched third-party org.apache.velocity .jar file with changes from https://github.com/apache/velocity-engine/pull/16 to resolve CVE-2020-13936.

### CODE REVIEW

File: lib/versions-complete.xml [hunk 1]
```diff
@@ -5466,12 +5466,12 @@
...
 			<library>
 				<file-name>com.liferay.portal.template.velocity.jar!org.apache.velocity.jar</file-name>
-				<version>1.6.4.LIFERAY-PATCHED-1</version>
+				<version>1.6.4-LIFERAY-PATCHED-2</version>
 				<project-name>org.apache.velocity</project-name>
 				<licenses>
 					<license>
-						<license-name>LGPL 2.1</license-name>
-						<license-url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt</license-url>
+						<license-name>Apache License, Version 2.0</license-name>
```
Comment (dantewang) [inline lib/versions-complete.xml:5473]: @mbowerman Is this change also built by `ant build-lib-versions`?

**Comment by dantewang** [comment]
@catarineaguiar Could you review the test results when they finish? Thanks!

File: lib/versions-complete.xml (hunk 1 above)
Comment (mbowerman) [reply lib/versions-complete.xml:5473]: Yes that change was generated by running the command `ant build-lib-versions`. I'm not sure why it made that change.

**Review by dantewang — APPROVED** [review]
@tinatian Code looks good. Waiting for test results and QA check.

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@mbowerman @dantewang no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/123354

**Pull request closed without merging**